docs/map-layer-configuration.md

```diff
@@ -1,65 +1,145 @@
+|--------------------|:---------:|---------|
+| **type**           | Indicator that the layer is a OpenStreetMap tile server layer, always `OSM` here  | `"type": "OSM"` |
 
 ## VECTOR
 
-| Property           | Mandatory | Meaning | Example |
-|--------------------|:---------:|---------|---------|
-| type               | x | Indicator that the layer is a vector layer, always `VECTOR` here  | `"type": "VECTOR"` |
-| url                | x  | The URL to the vector data resource (file) | `"url": "./static/data/2012_Earthquakes_Mag5.kml"` |
-| format             | x | The format of the data linked in `url` (either `KML` or `GeoJSON` ) | `"format": "KML"` |
-| selectable         | x | Boolean value, whether the features of the layer can be selected by click in order to display the attributes in a window | `"selectable": true` |
-| hoverable         | x | Boolean value, whether the features of the layer can be hovered in order to display an attribute (see `hoverAttribute`) in a tooltip  | `"hoverable": true` |
-| style              | x | Object to define a rendering style for the features of the layer  |  |
+| Property           | Meaning | Example |
+|--------------------|:---------:|---------|
+| **type**           | Indicator that the layer is a vector layer, always `VECTOR` here  | `"type": "VECTOR"` |
+| **url**            | The URL to the vector data resource (file) | `"url": "./static/data/2012_Earthquakes_Mag5.kml"` |
+| **format**         | The format of the data linked in `url` (either `KML` or `GeoJSON` ) | `"format": "KML"` |
+| selectable         | Boolean value, whether the features of the layer can be selected by click in order to display the attributes in a window | `"selectable": true` |
+| hoverable          | Boolean value, whether the features of the layer can be hovered in order to display an attribute (see `hoverAttribute`) in a tooltip  | `"hoverable": true` |
+| style              | Object to define a rendering style for the features of the layer  |  |
+
+## WFS
+
+| Property           |  Meaning  | Example |
+|--------------------|:---------:|---------|
+| **type**           | Indicator that the layer is a vector layer, always `WFS` here  | `"type": "WFS"` |
+| **url**            | The URL to the Web Feature Service (WFS) | `"url": "https://ows.terrestris.de/geoserver/osm/wfs"` |
+| **typeName**       | The name of the feature type | `"typeName": "osm:osm-fuel"`|
+| style              | Object to define a rendering style for the features of the layer  |  |
+| version            | The version of the WFS, defaults to `1.1.0` | `"version": "2.0.0"`|
+| maxFeatures        | Limits the amount of features that are returned | `"maxFeatures": 50`|
+| format             | The format that should be used, defaults to `GML3` |  `"format": "GML3"`|
+| selectable         | Boolean value, whether the features of the layer can be selected by click in order to display the attributes in a window | `"selectable": true` |
+
```

Comment:
`hoverable` is missing. Please copy the column from the the vector config.

Reply:
In the [WFS-function](https://github.com/meggsimum/wegue/blob/master/src/factory/Layer.js#L122-L194) there is no `hoverable` mentioned. Am I missing something?

Reply:
Oh, then it is missing for WFS layers at all. I opened an issue for that: https://github.com/meggsimum/wegue/issues/162

Reply:
alright, the we can later add the `hoverable` property to the docs